src/api/axiosInstance.ts

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import { getCookie, setCookie } from "@/utils/manageCookie";
+import { getNewToken } from "./getAuthApi";
+
+export const axiosInstance = axios.create({
+  baseURL: "https://bootcamp-api.codeit.kr/api/linkbrary/v1/",
+  headers: {
+    "Content-Type": "application/json",
+    withCredentials: true,
+  },
+});
+
+axiosInstance.interceptors.request.use(
+  (config) => {
+    const accessToken = getCookie("accessToken");
+    config.headers["Authorization"] = `Bearer ${accessToken}`;
+    return config;
+  },
+  (error) => {
+    return Promise.reject(error);
+  },
+);
+
+axios.interceptors.response.use(
```

Comment:
여기 인터셉터를 axiosInstance에 안붙여주신것 같아요!
```suggestion
axiosInstance.interceptors.response.use(
```

Reply:
레전드 똥멍충이짓을 했군요... ㅋㅋ ㅠㅠㅠㅠ 알려주셔서 감사합니다!!!!